### PR TITLE
feat(auth): option to disable strong password validation

### PR DESCRIPTION
**Pull request by siddhantk232**
Add option to disable strong password validation in `/-/auth/create-account/`.
This can be disabled using the `FASTN_DISABLE_STRONG_PASSWORD_CHECK` env var:

- `true`: disable validation
- `false` (or unset): use [`zxcvbn`](https://docs.rs/zxcvbn/2.2.2/zxcvbn/feedback/struct.Feedback.html) to check for weak passwords

Also removes some `dbg!` calls in `set_password.rs`.

### CODE REVIEW

File: integration-tests/_tests/04-multi-endpoint-test.test.ftd [hunk 1]
```diff
@@ -13,7 +13,7 @@ fastn.assert.eq(fastn.http_response["data"], "Hello, World!");
 ;; Mountpoint: /ftd/* -> Endpoint: http://fastn.com/ftd/*
 -- fastn.get: Fetching content from fastn.com
 url: /ftd/column/
-http-status: 308
+http-status: 301
```
Comment (amitu) [inline integration-tests/_tests/04-multi-endpoint-test.test.ftd:16]: Why this change? 307/308 are better replacements of 301/302.

**Comment by amitu** [comment]
This is because of Nandhini's comments? I am not yet sure if FifthTry should use this feature. I am okay with this being merged in general.

File: integration-tests/_tests/04-multi-endpoint-test.test.ftd (hunk 1 above)
Comment (siddhantk232) [reply integration-tests/_tests/04-multi-endpoint-test.test.ftd:16]: I did this because the CI was failing but not because of the changes introduced in this PR. I couldn't track down the change that caused it

Comment (amitu) [reply integration-tests/_tests/04-multi-endpoint-test.test.ftd:16]: Got it.

**Comment by siddhantk232** [comment]
> This is because of Nandhini's comments? I am not yet sure if FifthTry should use this feature. I am okay with this being merged in general.

Nope. I added this as a nice option to not have to come up with strong creative passwords when you're in a development environment. I remember you requesting this feature too

**Comment by amitu** [comment]
> to not have to come up with strong creative passwords when you're in a development environment

Got it. So this is a development feature. We as a convention add either `DANGER_` (if it is a prod variable that you should not set normally) or `DEBUG_` (for development/debug variables) as prefix. FASTN_DISABLE_STRONG_PASSWORD_CHECK
